tests/test_device.py

```diff
@@ -378,6 +378,63 @@ def test_sample_attribute_error(self, mock_device_with_paulis_and_methods):
             mock_device_with_paulis_and_methods.execute(queue, observables)
 
 
+class TestParameters:
+    """Test for checking device parameter mappings"""
+
+    @pytest.fixture
+    def mock_device(self):
+        with patch.multiple(
+            Device,
+            __abstractmethods__=set(),
+            operations=PropertyMock(return_value=["PauliY", "RX", "Rot"]),
+            observables=PropertyMock(return_value=["PauliZ"]),
+            short_name=PropertyMock(return_value="MockDevice"),
+            expval=MagicMock(return_value=0),
```

Comment:
Those should also work with normal `Mock`s, shouldn't they? I also posed myself the question what to use as default, but I concluded that I like it better to have the less powerful as default.

Reply:
I suppose so, but I wanted to keep it consistent with the previous tests :) Would be confusing for another developer to wonder why some mocks were different then others.